checkov/terraform/plan_parser.py

```diff
@@ -242,6 +242,28 @@ def _get_module_call_resources(module_address: str, root_module_conf: dict[str,
     return cast("list[dict[str, Any]]", root_module_conf.get("resources", []))
 
 
+def _get_provider(template: dict[str, Any]) -> dict[str, dict[str, Any]]:
+    """Returns the provider dict"""
+
+    provider_map = {}
+    provider_config = template.get("configuration", {}).get("provider_config", None)
```

Comment:
```suggestion
    provider_config = template.get("configuration", {}).get("provider_config")
``` u don't need the None, it will be None by default.